### PR TITLE
Bump openssh to 9.2

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

**Review by rawlingsj — DISMISSED** [review]

**Comment by dlorenc** [comment]
Rebased

**Comment by kaniini** [comment]
Needs secfixes data for CVE-2023-25136, which was fixed in 9.2!

**Comment by dlorenc** [comment]
> Needs secfixes data for [CVE-2023-25136](https://github.com/advisories/GHSA-w62j-g234-3f6f), which was fixed in 9.2!

Fixed!

**Review by rawlingsj — DISMISSED** [review]

File: openssh.yaml [hunk 1]
```diff
@@ -129,3 +121,13 @@ subpackages:
     dependencies:
       runtime:
         - openssh-keygen
+
+advisories:
+  CVE-2023-25136:
+    - timestamp: 2023-02-06T13:39:35.322093-05:00
+      status: fixed
+      fixed-version: 9.0_p1-r0
```
Comment (kaniini) [inline openssh.yaml:129]: This CVE was fixed in 9.2, not 9.0.

```suggestion
      fixed-version: 9.2_p1-r0
```

File: openssh.yaml [hunk 2]
```diff
@@ -129,3 +121,13 @@ subpackages:
...
+
+secfixes:
+  9.0_p1-r0:
```
Comment (kaniini) [inline openssh.yaml:132]: Likewise.

```suggestion
  9.2_p1-r0:
```

File: openssh.yaml (hunk 1 above)
Comment (dlorenc) [reply openssh.yaml]: Fixed!

**Review by kaniini — APPROVED** [review]

**Pull request merged**